lib/matplotlib/tests/test_colors.py

```diff
@@ -224,6 +225,17 @@ def test_SymLogNorm():
     assert_array_almost_equal(normed_vals, expected)
 
 
+@cleanup
+def test_SymLogNorm_colorbar():
+    """
+    Test un-called SymLogNorm in a colorbar.
+    """
+    norm = mcolors.SymLogNorm(0.1, vmin=-1, vmax=1, linscale=1)
+    fig = plt.figure()
+    cbar = mcolorbar.ColorbarBase(fig.add_subplot(111), norm=norm)
+    plt.close(fig)
```

Comment:
I think this `close` is unnecessary; the `@cleanup` decorator handles it, doesn't it?